src/pandroid/app/src/main/AndroidManifest.xml

```diff
@@ -2,10 +2,6 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
Removing this will break ELF/AXF loading on older devices (while still keeping it broken on newer devices)
Refer to https://discord.com/channels/1118695732958994532/1118702196851216404/1202987391611306044

Reply:
Fixed!